open_spiel/games/maedn.h

```diff
@@ -0,0 +1,281 @@
+// Copyright 2022 Thorsten Jungblut
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OPEN_SPIEL_GAMES_MAEDN_H_
+#define OPEN_SPIEL_GAMES_MAEDN_H_
+
+#include <array>
+#include <memory>
+#include <set>
+#include <string>
+#include <vector>
+
+#include "open_spiel/spiel.h"
+
+// An implementation of Mensch-Aergere-Dich-Nicht (see
+// https://de.wikipedia.org/wiki/Mensch_%C3%A4rgere_Dich_nicht)
+//
+// Rules used:
+// - start field must be cleared as soon as possible
+// - throwing out own pieces is not possible
+// - only one dice roll even if no move is possible except if dice roll was
+//   a six, in this case, same player may roll again
+// - pieces may jump over each other on four final fields
+//
+// Parameters:
+//   none yet
```

Comment:
Looks like there are two params now: num players and two players opposite. I'd suggest moving the description of params from the cc file to here. There's nothing technically wrong with it there, but here is preferable for two reasons:
- Consistency: this is the pattern followed by most other games and it makes it easier to look up info
- Editing/inspecting params: putting the comments there can spread out the params and make it harder to locate/edit/inspect them

Reply:
You're right, i've moved the comments accordingly.